plugins/linuxv2/model/l3/l3.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package l3;
+
+/* static ip route */
+message LinuxStaticRoute {
```

Comment:
question - I miss name here, will it be generated from other fields? Same for arp.

Reply:
@milanlenco mentioned in our last discussion that he chose to remove the name field for things like routes, because we don't set it to southbound in any way and the name does not actually represent the route and is not included in key either

Reply:
For ARPs and routes I chosed what we do in VPP l3 plugin - not naming things, but identifying them by their primary fields. This is for several reasons:
 1. dumping is simplified - you can build value + key from the dumped data without needing support for metadata (like tags, which in Linux are not available)
2. arps and routes may be created externally (e.g. dynamic entries), but the same route/arp by value will have the same ID, regardless of whether it was configured by the agent or came from somewhere else - thus the scheduler will be able to compare and match the same items (thus not trying to configure the same twice)
3. With name as the only primary key, you can create two entries which are effectively the same (thus ending with add-error for the second one) - I wanted to prevent that from happening
4. In many cases you will end up building the name from the primary fields anyway

I would say that configuration items should be logically named only when:
1. there can be multiple items of the same value (i.e. no primary keys in the value) - e.g. ACLs
2. there are too many fields needed to include in the key for unique identification (and tagging is supported)
2. there is more to the item than just it's value - e.g. interface "goes" somewhere
3. the item is already "logical" - e.g. bridge domain
etc.